Azure-Samples/Azure-Time-Series-Insights
# Delete JS App

## Purpose

Delete unneeded file.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
1. Make sure the AAD auth information matches (redirect URI matches, client ID)
1. Serve the html in javascript-tsi-ga-sample
1. Verify no breaking changes
```

## What to Check
Verify that the following are valid
* All JS is correctly rendering
* All data is returned as intended

## Other Information
